### PR TITLE
Run subsystem schema tests by default

### PR DESCRIPTION
**Pull request by kabir**
https://issues.jboss.org/browse/WFLY-5658

This and https://github.com/wildfly/wildfly/pull/8396 will need to be merged before https://github.com/wildfly/wildfly-core/pull/1250 is in a core release

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -216,7 +216,7 @@
         <version.org.syslog4j>0.9.30</version.org.syslog4j>
         <version.org.wildfly.build-tools>1.1.0.Final</version.org.wildfly.build-tools>
         <version.org.wildfly.checkstyle-config>1.0.4.Final</version.org.wildfly.checkstyle-config>
-        <version.org.wildfly.core>2.0.1.Final</version.org.wildfly.core>
+        <version.org.wildfly.core>2.0.2.Final-SNAPSHOT</version.org.wildfly.core>
```
Comment (bstansberry) [inline pom.xml]: This won't work. :)

It doesn't seem to be needed though. I was able to revert this line and do a build, at least through the smoke tests.

Comment (kabir) [inline pom.xml]: oops :-)

**Comment by bstansberry** [comment]
Looks good. Thanks.

I've put the 11.x label on this as I don't thinks it's needed for 10.

Last week there were some discussions around this general topic on the EAP side. If those lead to a desire to get this in the EAP branch for 7.0, then we can do a JBEAP and port this and the core PR.

**Comment by bstansberry** [comment]
@kabir Is this still wanted? I forget where we came out re: running these tests.

**Comment by bstansberry** [comment]
@kabir Is this still wanted? I forget where we came out re: running these tests.

**Pull request closed without merging**